README.md

```diff
@@ -40,64 +40,141 @@
 </div>
 
 ## Table of Contents
-- [API](#api)
-- [Installation](#installation)
 - [Development](#development)
-- [Environment](#environment)
+- [API](#api)
 - [Testing](#testing)
 - [Commands](#commands)
 - [Code style](#code-style)
 - [Contributors](#contributors)
 - [See Also](#see-also)
 
-## API
+## Getting Started
 
-If you want to build on the API for personal use, take a look through the work done in the [Migrate to new APIs project](https://github.com/resonatecoop/stream/projects/5#card-59829409) and consider checking the [backlog in our community forum](https://community.resonate.is/c/platform/52). 
+### Quickstart - no HTTPS
+
+Quick-n-dirty instructions to get the player up and running on your computer using http and pointing to the existing production API (see [API](#api) to learn more about the API). 
+Assumes the latest version of [node.js](https://nodejs.org/).
 
-## Installation
+_Stuck or curious about the roadmap? Ask for help in the [forum](https://community.resonate.is/t/development-team/1724)_. 
 
-First, make sure you have the latest version of [node.js](https://nodejs.org/)
+Clone the repo and `cd` into it:
 
-To use this project you also need [lerna](https://github.com/lerna/lerna). Lerna is a tool for managing JavaScript projects with multiple packages. .
+```sh
+git@github.com:resonatecoop/stream.git
+cd stream
+```
+
+Install [lerna](https://github.com/lerna/lerna). Lerna is a tool that optimizes the workflow around managing multi-package repositories with git and npm. 
 
 ```sh
 npm i lerna -g
 ```
 
-Then you should install dependencies using the bootstrap command.
+Install dependencies: 
 
 ```sh
 lerna bootstrap
 ```
 
-## Development
+Create your env file:
 
-### Beta app
+```sh
+cp beta/.env.example beta/.env
+```
+
+You may point your `.env` file to the production API:
+
+```
+API_DOMAIN=api.resonate.coop
```

Comment:
maybe have different env examples for each use case?

Reply:
some things might change when https://github.com/resonatecoop/stream/pull/100 is merged ... it would make more sense to me to revisit this issue after that

Reply:
@auggod Are these environment variables going to change when  #100 is merged?

Reply:
@whatSocks #100 is merged now. Let's update the default env variables as part of this PR so we can streamline this for new folks. We can keep updating that file as they change. Then all we need to do is ask people to copy it.